frontend/pipeline.yml

```diff
@@ -207,7 +207,7 @@ build:frontend:docker:
         - frontend/junit/results.xml
     when: always
   tags:
-    - hetzner-amd-beefy
+    - k8s
```

Comment:
By moving to k8s runner, you have to define the DOCKER variables:

```
  variables:
    DOCKER_HOST: tcp://docker:2375
    DOCKER_TLS_CERTDIR: ""
```
And also add the `HEALTHCHECK_TCP_PORT` variable to the Docker service:
```
  services:
    - name: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/docker:${DOCKER_VERSION}-dind
      alias: docker
      variables:
        HEALTHCHECK_TCP_PORT: "2375"
```